kythe/go/serving/xrefs/xrefs_test.go

```diff
@@ -1331,10 +1331,18 @@ func TestCrossReferences(t *testing.T) {
 	if err := testutil.DeepEqual(&xpb.CrossReferencesReply_Total{
 		Definitions: 1,
 		References:  2,
+		RefEdgeToCount: map[string]int64{
+			"/kythe/edge/ref": 2,
+		},
 	}, reply.Total); err != nil {
 		t.Error(err)
 	}
-	if err := testutil.DeepEqual(&xpb.CrossReferencesReply_Total{}, reply.Filtered); err != nil {
+	wantFiltered := &xpb.CrossReferencesReply_Total{
+		RefEdgeToCount: map[string]int64{
+			"/kythe/edge/ref": 0,
```

Comment:
Do we want these superfluous zero counts?

Reply:
I added code to remove them.